eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -150,6 +155,7 @@ jobs:
   variables:
   - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
     - group: AzureDevOps-Artifact-Feeds-Pats
+    - group: DotNet-VSTS-Infra-Access
```

Comment:
nit: maybe only inject this when `enableIBCOptimization` is true just as a defense-in-depth